packages/database/src/core/util/Tree.ts

```diff
@@ -155,7 +155,7 @@ export class Tree<T> {
    * @return {boolean} true if the action callback returned true.
    */
   forEachAncestor(
-    action: (tree: Tree<T>) => void,
+    action: (tree: Tree<T>) => any,
```

Comment:
since return value of action is used in a if statement, it needs to return a type. Should it be boolean?